packages/components/src/DateTimePickers/DateTimeView/DateTimeView.scss

```diff
@@ -0,0 +1,16 @@
+.title {
+    width: 100%;
```

Comment:
The width and height 100% in all views seems to be something related to ViewLayout. You do that to fit the container.
What about removing all those, and adding a style in ViewLayout to make the content fit the container ?

Example
```
.body {
    > * {
        width: 100%;
        height: 100%;
    }
}
```

Reply:
&#10003;